tests/tensor_search/test_add_documents.py

```diff
@@ -358,7 +358,7 @@ def test_add_documents_validation(self):
             add_res = tensor_search.add_documents(
```

Comment:
We could consider removing these tests since the one below testing `replace` covers it